README.md

```diff
@@ -100,6 +100,24 @@ npm run prod
 
 _Although the build of the client is production-ready, Nuxt's own HTTP server should never be exposed directly to the internet and a proper hosting tool like Nginx should always be used instead._
 
+</br>
+
+## Running Docker Container
+
+---
```

Comment:
No needed as the header already adds a divisor

Reply:
Noted.